test/round-tracker.test.js

```diff
@@ -221,8 +221,9 @@ describe('Round Tracker', () => {
   describe('startRoundTracker', () => {
     it('detects the current round', async function () {
       this.timeout(TIMEOUT_WHEN_QUERYING_CHAIN)
-      const { sparkRoundNumber, close } = await startRoundTracker(pgPool)
-      close()
+      const controller = new AbortController()
+      const { sparkRoundNumber } = await startRoundTracker(pgPool, controller.signal)
+      controller.signal.abort()
```

Comment:
This is not going to work. When `await startRoundTracker` throws an error, you will never reach the second line to abort the controller/signal.

I am proposing a slightly different implementation: https://github.com/filecoin-station/spark-api/pull/305

Reply:
Right! Will update once 305 lands 👍